Vungle/VungleRouter.m

```diff
@@ -668,6 +660,48 @@ - (void)vungleDidCloseAdWithViewInfo:(VungleViewInfo *)info placementID:(NSStrin
     }
 }
 
+- (void)vungleTrackClickForPlacementID:(nullable NSString *)placementID {
+    id<VungleRouterDelegate> targetDelegate;
+
+    if ([placementID isEqualToString:self.bannerPlacementID]) {
+        for (int i = 0; i < self.bannerDelegates.count; i++) {
+            if ((BannerRouterDelegateState)[[self.bannerDelegates[i] valueForKey:kVungleBannerDelegateStateKey] intValue] == BannerRouterDelegateStatePlaying) {
+                targetDelegate = [self.bannerDelegates[i] objectForKey:kVungleBannerDelegateKey];
+            }
+        }
+    } else {
+        targetDelegate = [self.delegatesDict objectForKey:placementID];
```

Comment:
is there any possibility that placementID here will be nil?

Reply:
@crbinsf Got it, I will add the check if placementID is nil. Thanks. :)